src/components/GroupSystems/GroupImmutableSystems.js

```diff
@@ -0,0 +1,367 @@
+      )}
+      {removeHostsFromGroupModalOpen && (
+        <RemoveHostsFromGroupModal
+          isModalOpen={removeHostsFromGroupModalOpen}
+          setIsModalOpen={setRemoveHostsFromGroupModalOpen}
+          modalState={currentSystem}
+          reloadTimeout={1000}
+          reloadData={() => {
+            if (calculateSelected() > 0) {
+              dispatch(selectEntity(-1, false));
+            }
+
+            inventory.current.onRefreshData({}, false, true);
+          }}
+        />
+      )}
+      {updateDevice && (
+        <AsyncComponent
+          appName="edge"
+          module="./UpdateDeviceModal"
+          navigateProp={useNavigate}
+          locationProp={useLocation}
+          notificationProp={notificationProp}
+          paramsProp={useParams}
+          updateModal={updateModal}
+          setUpdateModal={setUpdateModal}
+          refreshTable={() => true}
+        />
+      )}
+      {!addToGroupModalOpen && (
+        <InventoryTable
+          columns={(columns) => mergeColumns(columns)}
+          hideFilters={{ hostGroupFilter: true }}
+          // getEntities={entities}
+          getEntities={customGetEntities}
+          tableProps={{
+            isStickyHeader: true,
+            variant: TableVariant.compact,
+            canSelectAll: false,
+            actionResolver: (row) => [
+              {
+                title: (
+                  <ActionDropdownItem
+                    requiredPermissions={REQUIRED_PERMISSIONS_TO_MODIFY_GROUP(
+                      groupId
+                    )}
+                    noAccessTooltip={NO_MODIFY_GROUP_TOOLTIP_MESSAGE}
+                    onClick={() => {
+                      setCurrentSystem([row]);
+                      setRemoveHostsFromGroupModalOpen(true);
+                    }}
+                  >
+                    Remove from group
+                  </ActionDropdownItem>
+                ),
+                style: {
+                  padding: 0, // custom component creates extra padding space
+                },
+              },
+              {
+                title: (
+                  <ActionDropdownItem
+                    isAriaDisabled={
+                      deviceData && !deviceData.find((obj) => obj === row.id)
+                    }
+                    requiredPermissions={REQUIRED_PERMISSIONS_TO_MODIFY_GROUP(
+                      groupId
+                    )}
+                    noAccessTooltip={NO_MODIFY_GROUP_TOOLTIP_MESSAGE}
+                    onClick={() => {
+                      setCurrentSystem([row]);
+                      navigate(`/insights/inventory/${row.id}/update`);
+                    }}
+                  >
+                    Update
+                  </ActionDropdownItem>
+                ),
+                style: {
+                  padding: 0, // custom component creates extra padding space
+                },
+              },
+            ],
+          }}
+          actionsConfig={{
```

Comment:
The buttons do not have a margin between each other.

<img width="837" alt="Screenshot 2023-11-23 at 12 16 38" src="https://github.com/RedHatInsights/insights-inventory-frontend/assets/31385370/2ae9abdd-5b03-4250-986d-f413877ef335">

Can you please also double-check whether the button must be hidden under three-dots kebab button? Usually, only one action is available on top of the primary toolbar and other are hidden as secondary actions.

Reply:
by the mockup definition should be 2 and the enable/disable is handle by the device selection and we send the update to the last version

Reply:
fixed the layout